src/Microsoft.AspNet.Authentication.Facebook/Notifications/FacebookAuthenticationNotifications.cs

```diff
@@ -23,16 +24,33 @@ public FacebookAuthenticationNotifications()
         /// <summary>
         /// Gets or sets the function that is invoked when the Authenticated method is invoked.
         /// </summary>
-        public Func<FacebookAuthenticatedContext, Task> OnAuthenticated { get; set; }
+        public new Func<FacebookAuthenticatedContext, Task> OnAuthenticated { get; set; }
```

Comment:
Ok I don't want to cause more feature creep. I will take a look at it this week and see if there's anything we can do to simplify things.